instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/chekboxandradio/SrCheckBoxAndRadioFieldsAllowTest.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sdk.integration.sessionreplay.chekboxandradio
+
+import com.datadog.android.privacy.TrackingConsent
+import com.datadog.android.sdk.integration.sessionreplay.SessionReplayRadioCheckboxFieldsActivity
+import com.datadog.android.sdk.integration.sessionreplay.SrTest
+import com.datadog.android.sdk.rules.SessionReplayTestRule
+import com.datadog.android.sdk.utils.SR_PRIVACY_LEVEL
+import com.datadog.android.sessionreplay.SessionReplayPrivacy
+import org.junit.Rule
+import org.junit.Test
+
+internal class SrCheckBoxAndRadioFieldsAllowTest :
```

Comment:
let's use `SessionReplay` in naming instead of `Sr`.

using just `Sr` will give many false-positives if text search is performed.

Reply:
Actually wanted to do this but if I do it I need to do it for all the classes there and some of them will be way too long 😮‍💨